src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -475,18 +478,16 @@ const Wrapper = styled.aside`
       width: 336px;
       height: 482px;
       position: unset;
-      border: 1px solid rgba(209, 209, 209, 0.5);
       box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.15);
```

Comment:
```suggestion
      box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.15);
```

The design file has a slightly large box-shadow (but uses the wrong colour, so this adjusts it back to black).